searchcommon/src/vespa/searchcommon/common/schema.cpp

```diff
@@ -131,60 +131,48 @@ Schema::Field::operator!=(const Field &rhs) const
 
 Schema::IndexField::IndexField(vespalib::stringref name, DataType dt)
     : Field(name, dt),
-      _prefix(false),
-      _phrases(false),
-      _positions(true),
-      _avgElemLen(512)
+      _avgElemLen(512),
+      _experimental_posting_list_format(false)
 {
 }
 
 Schema::IndexField::IndexField(vespalib::stringref name, DataType dt,
                                CollectionType ct)
     : Field(name, dt, ct),
-      _prefix(false),
-      _phrases(false),
-      _positions(true),
-      _avgElemLen(512)
+      _avgElemLen(512),
+      _experimental_posting_list_format(false)
 {
 }
 
 Schema::IndexField::IndexField(const std::vector<vespalib::string> &lines)
     : Field(lines),
-      _prefix(ConfigParser::parse<bool>("prefix", lines)),
-      _phrases(ConfigParser::parse<bool>("phrases", lines)),
-      _positions(ConfigParser::parse<bool>("positions", lines)),
-      _avgElemLen(ConfigParser::parse<int32_t>("averageelementlen", lines))
+      _avgElemLen(ConfigParser::parse<int32_t>("averageelementlen", lines)),
+      _experimental_posting_list_format(ConfigParser::parse<bool>("experimentalpostinglistformat", lines))
```

Comment:
This will likely break both upgrade and downgrade due to not having default values.

Reply:
Fixed